shell/browser/api/electron_api_app.cc

```diff
@@ -1090,15 +1104,22 @@ bool App::RequestSingleInstanceLock() {
 
   auto cb = base::BindRepeating(&App::OnSecondInstance, base::Unretained(this));
 
+  v8::Isolate* isolate = args->isolate();
+  v8::Local<v8::Value> data_arg = args->PeekNext();
+  blink::CloneableMessage additional_data_message;
+  if (!data_arg.IsEmpty()) {
+    SerializeV8Value(isolate, data_arg, &additional_data_message);
+  }
```

Comment:
I'll stick to the serializer for now because I get the following error:

```
error: no member named 'FromV8' in 'gin::Converter<blink::CloneableMessage>'
return Converter<T>::FromV8(isolate, input, result);
```

Reply:
You will need to include the respective converter headers, for this case it is `shell/common/gin_converters/blink_converter.h`